rules/xcodeproj.bzl

```diff
@@ -44,6 +45,20 @@ def _srcs_info_build_files(ctx):
 
     return [path]
 
+def _xcodeproj_aspect_collect_hmap_paths(deps, target, ctx):
+    hmap_paths = []
+    kind = ctx.rule.kind
+    is_library = kind == "objc_library" or kind == "swift_library"
```

Comment:
why is the `kind` check necessary? shouldn't we be able to just look for the `HeaderMapInfo` provider without caring what rule it was instantiated in?

Reply:
I thought not doing this check will bring unrelated hmaps here, but after testing this is not the case. Will remove this check. Thanks!